packages/cli/src/commands/test.ts

```diff
@@ -100,6 +101,9 @@ export default class Test extends AuthCommand {
       description: 'Update any snapshots using the actual result of this test run.',
       default: false,
     }),
+    retries: Flags.integer({
+      description: 'How many times to retry a failing test run.',
```

Comment:
TODO: We should document the max number of retries here

Reply:
Done ✅ 

@ferrandiaz as discussed, I added the max number of retries to the help output. Also added in the default there.

The brackets in front are to be consistent with the formatting we already have for defaults.